src/views/little_machine/little_machine_index_view.js

```diff
@@ -0,0 +1,26 @@
+import BaseView from '../base_view';
+import LittleMachineIndexTemplate from '../../templates/little_machine/index.html';
+import LittleMachineItemView from './little_machine_item_view';
+import LittleMachineResults from '../../collections/little_machine_results';
+import LittleMachineAdapter from '../../adapters/little_machine_adapter';
+
+export default BaseView.extend({
+  template: LittleMachineIndexTemplate,
+
+  initialize () {
+    this.little_machine = LittleMachineAdapter;
```

Comment:
Yeah, I'm ignore ESLint for now as I shuffle things around. I'll make sure it's not complaining before I remove the [wip].